src/bloom/utils.rs

```diff
@@ -584,13 +608,19 @@ mod tests {
         let rand_prefix = random_prefix(7);
         // 1 in every 1000 operations is expected to be a false positive.
         let expected_fp_rate: f64 = 0.001;
+        let expected_tightening_ratio: f64 = 0.5;
```

Comment:
```suggestion
        let tightening_ratio: f64 = 0.5;
```

We are not asserting against this. so there is no need to call it "expected"

Reply:
Will change accordingly